src/pspm_load1.m

```diff
@@ -74,14 +74,18 @@
   warning('ID:missing_data', 'Save failed, no data provided'); return;
 end
 
-errmsg = sprintf('Data file %s is not a valid PsPM file:\n', fn);
 
 % canonicalise file name
-[pth, filename, ext] = fileparts(fn);
-if isempty(ext)
-  ext = '.mat';
+if ischar(fn)
```

Comment:
I have not checked, but if necessary, shall I check if `fn` is checked in every function? I checked some and they used `pspm_load_channel` so I can understand they did not check `fn` by them own, but it may be useful to check every function.